cerberus/validator.py

```diff
@@ -1038,8 +1058,17 @@ def _validate_keyschema(self, schema, field, value):
     def _validate_readonly(self, readonly, field, value):
         """ {'type': 'boolean'} """
         if readonly:
-            self._error(field, errors.READONLY_FIELD)
-            return True
+            if not self.is_normalized:
+                self._error(field, errors.READONLY_FIELD)
+                return True
+            # If the document was normalized (and therefore already been
+            # checked for readonly fields), we still have to return True
+            # if an error was filed if readonly is a priority rule.
+            if 'readonly' in self.priority_validations:
```

Comment:
No, I did not think about Eve at all when implementing this. There is this "configuration" for priority validations, so I thought it's best to reuse it instead of hardcoding the fact. But thinking about that again it's special anyway because returning True is just something priority validations do. I will remove that check.